configure.py

```diff
@@ -945,7 +945,7 @@ def is_cuda_compatible(lib, cuda_ver, cudnn_ver):
 
 def set_tf_tensorrt_version(environ_cp):
   """Set TF_TENSORRT_VERSION."""
-  if not is_linux():
```

Comment:
This test can clearly be improved.

`if not (is_linux() or is_windows()):`

Reply:
Fixed